src/components/pagination/pagination.styles.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { css } from '@emotion/react';
+import { logicalCSS, euiScrollBarStyles } from '../../global_styling';
+import { UseEuiTheme } from '../../services';
+
+export const euiPaginationStyles = (euiThemeContext: UseEuiTheme) => {
+  const { euiTheme } = euiThemeContext;
+
+  return {
+    // Base
+    euiPagination: css`
+      ${euiScrollBarStyles(euiThemeContext)};
+      display: flex;
+      align-items: center;
+      ${logicalCSS('overflow-y', 'hidden')};
+      ${logicalCSS('overflow-x', 'auto')};
```

Comment:
This needs to use `logicalCSSWithFallback`, as overflow-block/size is currently only supported by Firefox
```suggestion
      ${logicalCSSWithFallback('overflow-y', 'hidden')};
      ${logicalCSSWithFallback('overflow-x', 'auto')};
```